meetings/2022-03-09.md

```diff
@@ -0,0 +1,134 @@
+* Wouter: I think it does make sense but I'm not sure we're talking about exactly the same thing. I think grants are pretty clearly the domain of the authorization agents. There should be also some server delivering access tokens based on permissions. But if we take the grants delivered by the authz agent, if they are the single source of truth (thereby being source of truth for more than one resource), then there needs to be a way to sync that truth with/in the Solid ACL auxiliary resources. Does the server performing that sync needs to be the same server as this UMA server (which needs a lot less informations and permissions than the server that needs to perform the sync).
+* Matthieu: By UMA, you mean AS?
+* Wouter: I think what we understand to be AS is a single entity that has two roles 1. deliver an Access Token (by pushing VCs or exchanging an ID token) which only requires knowledge of the UMA ticket 2. syncing the grants set by the authorization agent onto the resources (needs a lot of permissions or implicit trust).
+* Matthieu: I'm not sure about syncing Data Grants into auxiliary ACL resources or on tight coupling every authz scenario to an Authorization Agent. I don't think those two would translate well. I don't see them as mutually exclusive.
+* Wouter: But than we need to find a way to decide which one has priority. Which one acts as the source of truth. If there is an ACL stating that X doesn't have access to a resource yet I'm an owner and i issue a data grant that X has access. What's the truth.
+* Elf: I agree that we need to agree on what is the single source of truth. Probably ACP/WAC/ACL could not necessarily cohabit with data grants. Myself I think that some storage type would only work with data grants. Specifically if pushing a claim could resolve in changing an ACL on the AS. What if you used a notification stream to get the AS updating the ACL?
+* Matthieu: What kind of claim are you talking about?
+* Elf: The client is pushing claims. If we don't have an out of band way to propagate delegation, for the client to get any access, they'd need to present a verifiable credential that proves a user authorized that client to have access. If the end user of the client is not the resource owner, they would issue a delegated data grant. Before that claim, no one would know that the user has authorized a client. That would be reflected in updating the ACL on that server.
+* ...: If it's happening out of band via the authz agent, client wouldn't need to push it as a claim
+* Matthieu: This whole thread is quite confusing.
+* Wouter: Yes, I think it's a lot of inter-mixed decisions or a lot of different aspects that influence each other.
+* Elf: We have a use case about permissioning applications. The end user is not the resource owner and wants to access and authorise a client. How currently would you resolve that use case with raw ACP? Say the end user only gets read/write from the ACL but it's client.
```

Comment:
@elf-pavlik -- `Say the end user only gets read/write from the ACL but it's client.` doesn't make enough sense for me to correct.

Reply:
```suggestion
* Elf: We have a use case about permissioning applications. The end user is not the resource owner and wants to access and authorise a client. How currently would you resolve that use case with raw ACP? Say the end user gets read/write from the ACL but its client only gets read.
```